auxiliary_tools/cdat_regression_testing/base_run_script.py

```diff
@@ -27,8 +27,25 @@
 )
 from e3sm_diags.run import runner
 
+# Get the current machine's configuration info.
+MACHINE_INFO = MachineInfo()
+MACHINE = MACHINE_INFO.machine
+
+if MACHINE not in [
+    "anvil",
+    "chrysalis",
+    "compy",
+    "pm-cpu",
+    "cori-haswell",
+    "cori-knl",
+]:
+    raise ValueError(f"e3sm_diags is not supported on this machine ({MACHINE}).")
+
 # The location where results will be stored based on your branch changes.
-BASE_RESULTS_DIR = "/global/cfs/cdirs/e3sm/www/cdat-migration-fy24/"
+if MACHINE in ["chrysalis", "anvil"]:
+    BASE_RESULTS_DIR = "/lcrc/group/e3sm/public_html/cdat-migration-fy24/"
+elif MACHINE in ["cori-haswell", "cori-knl", "pm-cpu"]:
+    BASE_RESULTS_DIR = "/global/cfs/cdirs/e3sm/www/cdat-migration-fy24/"
```

Comment:
Updated the `base_run_script` to support LCRC too because perlmutter is down right now.